### ISSUE TITLE
import fails

### ISSUE THREAD

**Issue by rakool** [rakudo/rakudo#4122]
This is Rakudo version 2020.07-66-g2db92e04f built on MoarVM version 2020.07-16-g03d3e43fa
implementing Raku 6.d.

Linux shrew 4.19.0-12-amd64 #1 SMP Debian 4.19.152-1 (2020-10-18) x86_64 GNU/Linux

```import``` doesn't find ```load```ed file.
```
#!/usr/bin/env raku
use lib $?FILE.IO.parent(1);

#use MyMod;     # This works.

load MyMod; 
import MyMod;   # Fails: "Could not find module MyMod …" 

A.new.raku.say;   # output:  MyMod::A.new(a => "a")

=begin another-file
# MyMod.rakumod in same directory
unit module MyMod; 
class A is export { has $.s = 'a'}
=end another-file  

```
Closed 3407 and 2229  come up on search of 'import'.

**Comment by jnthn** [comment]
> load MyMod; 

There is no `load`; did you mean `need`?

**Comment by rakool** [comment]
Thank you!
I did, I did.    I meant `need`!   This was part of a string of incantations where voodoo may taint things.  A _clear_ problem on a tangent from a weird scoping problem.  (Which made me think of your work on macros as being a less likely work around than simple textual macro which would not absorb any error in the grammar.)   So I was wrong but perhaps there is still a Raku  failure here.

I just discovered in` CompUnit::Repository`, there is a  `method load ((IO::Path:D $file --> CompUnit:D)`.  It does not seem correct  by the doc's that I would invoke that. (?)

Then should the error come from `load` not being found?   That would have set me righter.
```
unit$ raku -c  t.t
===SORRY!=== Error while compiling /priv/rir/Repo/try/unit/t.t
Could not find module MyMod to import symbols from
at /priv/rir/Repo/try/unit/t.t:7
------> import MyMod⏏;   # Fails: "Could not find module MyMo

```
